mlir/test/Integration/Dialect/Complex/CPU/correctness.mlir

```diff
@@ -189,8 +186,8 @@ func.func @entry() {
     // CHECK-NEXT:  0
     // CHECK-NEXT:  0
     (0.0, 0.0), (-1.0, 0.0),
-    // CHECK-NEXT:  -nan
-    // CHECK-NEXT:  -nan
```

Comment:
[nit] You could add a note (for our future selves) referring to: https://github.com/llvm/llvm-project/issues/58531. For example: 
```
Ignoring the sign of nan as that can't be tested in platform agnostic manner. See: #58531
```